docs/validator.md

```diff
@@ -20,6 +20,61 @@ dbmate --url "sqlite:validator_database.db" up
 python neurons/validator.py --netuid 269 --subtensor.network test --wallet.name VALIDATOR_WALLET --wallet.hotkey VALIDATOR_HOTKEY --logging.debug --organic True --neuron.epoch_length 5 --axon.port AXON_PORT --dht.port 6945
 ```
 
+### Selling your bandwidth
+
+#### Managing access
+
+To manage access to the your api server and sell access to anyone you like, using the storb cli tool is the easiest way.
+
+```bash
+storb --help
+```
+
+Shows all the commands and should give self explanatory instructions.
+
+You can also do
+
+```bash
+storb some-command --help
+```
+
+To get more info about that command!
+
+#### Examples
+
+For example:
+
+```bash
+storb create-key --balance 10 --rate-limit-per-minute 60 --name test
+```
+Creates a test key with a balance of 10 (which corresponds to 10 requests), a rate limit of 60 requests per minute = 1/s, and a name 'test'.
+
+<!-- **Recommend values:**
+- Balance: Depends on how much you want to sell! Each credit is a image (so a balance of 1000 will allow 1000 images to be generated)
+- Rate limit: I would recommend a rate limit of ~20/minute for casual users trying out the API, and around ~120/minute for production users
+- Name: Just for you to remember who you want to use that key :) -->
```

Comment:
reword this to fit our subnet

Reply:
im just gonna remove it - the params are pretty self-explanatory anyway.